### ISSUE TITLE
Migration does not update sequences 

### ISSUE THREAD

**Issue by qury** [benedmunds/CodeIgniter-Ion-Auth#1167]
Hi
I've noticed that when setting up Ion Auth on PostgreSQL 10.1, i have trouble adding users and groups using the ion auth.

The problem seems to be with the id fields the sequences that drive these fields are not incremented properly.

I've attached a diff file that has a fix for it.
(sorry i do not yet know how to do pull request yet)

[migration.fix.txt](https://github.com/benedmunds/CodeIgniter-Ion-Auth/files/1519645/migration.fix.txt)

**Comment by benedmunds** [comment]
What if we just create these fields as SERIALS instead of as MEDIUMINTs?

**Comment by qury** [comment]
Hi Ben

I did some more testing and the problem is with adding the administrator user with an ID value 1.
When i try to add the second user the DB is trying to assign the next value from the sequence which is 1 and ends up with a unique key violation.

The SQL file does not have the id fields in the insert statement, so it works fine.

So the solution is to remove the 'id' fields from the insert data where the id is auto-incremented.

I've manged to figure out how to do a pull request :)  #1168
